docs/man/readtags.1.rst

```diff
@@ -172,22 +173,42 @@ missing field is #f, so we could deal with missing fields:
                          (or (eq? $language "Python")\
                              (not $language)))' -l
 
-The ``member`` operator is used to deal with lists. Currently lists can't be
-built by hand, and they are only generated for the ``inherits:`` field. For
-example:
+``#/PATTERN/`` is for the case when string predicates (``prefix?``, ``suffix``,
+and ``substr?``) are not enough. You can use "Posix extended regular expression"
+as PATTERN.
 
 * List all tags inherits from the class "A":
 
   .. code-block:: console
 
-     $ readtags -Q '(member "A" $inherits)' -l
+     $ readtags -Q '(#/(^|, )A(,|$)/ $inherits)' -l
+
+Here ``$inherits`` is a comma-separated class list like "A, B, C", "Z, A", "P, A,
+Q", or just "A". The tags file may have tag entries that has no ``inherits:``
+field. In that case ``$inherits`` is #f, and the regular expression matching
+raises an error, since it works only for strings. To avoid this problem:
+
+* Safely list all tags inherits from the class "A":
+
+  .. code-block:: console
+
+     $ readtags -Q '(and $inherits (#/(^|, )A(,|$)/ $inherits))' -l
 
-WARNING: It's planned to remove the ``member`` operator, and replace this
-technique with regular expression matching. By then, ``$inherits`` will not be
-a list, but the value of the ``inherits:`` field as it is.
+
+* Safely list all tags inherits from the class "A" or "a":
+
+  .. code-block:: console
+
+     $ readtags -Q '(and $inherits (#/(^|, )A(,|$)/i $inherits))' -l
 
 Run "readtags -H filter" to know about all valid functions and variables.
 
+NOTE: The above regular expression pattern for inspecting inheritances is just an
+example to show how to use ``#/PATTERN/`` expression.  Tags file generators have
+no consensus about the format of ``inherits:``.  Even parsers in ctags have no
+consensus. Handling such language-specific aspects is inevitable in a client tool
```

Comment:
Just a nitpicking: I've been careful when using the word "client tool" in this manpage. You can see it only appears once, and it's about using the `-E` option, i.e., about reading the output by a machine, not by human. But here, even interactive use needs to be aware of the format of the `inherits:` field, so I'd suggest change this sentence to:

```
Noticing the format of the ``inherits:`` field of specific languages is needed for such queries.
```